suites/pacific/cephfs/tier-1/mirror/02_deploy_fs_mirror/07_add_mds_daemons/override.yaml

```diff
@@ -0,0 +1,17 @@
+tests:
+  - test:
+        config:
+          command: apply
+          service: mds
+          base_cmd_args: # arguments to ceph orch
+            verbose: true
+            pos_args:
+              - cephfs # name of the filesystem
+            args:
+              placement:
+                nodes:
+                  - node4
+                  - node5
```

Comment:
This is a unique test and is to the same as `test mds using labels`. Could you please create a fragment for this test? It could be `add mds using node placement`